app/src/main/java/ch/epfl/qedit/backend/location/LocationService.java

```diff
@@ -0,0 +1,26 @@
+package ch.epfl.qedit.backend.location;
+
+import android.location.LocationListener;
+
+/**
+ * This interface is used to request location updates from the phone. It is an interface, so that it
+ * can be mocked for the tests.
+ */
+public interface LocationService {
+    /**
+     * Subscribes to the location service using the given location listener. The location updates
+     * should be done at the given interval (in milliseconds), if possible.
+     *
+     * @param listener the listener that will be subscribed.
+     * @param interval the interval at which updates should be received.
```

Comment:
I don't quite understand this parameter (`interval`)... Furthermore, the function is called `subscribe` and you want to send updates but where is the function for updates ?

Reply:
The listener is the one to receive the updates. The `interval` determines the interval at which we would ideally like to receive the updates.